lib/kitchen/driver/vagrant.rb

```diff
@@ -556,7 +556,7 @@ def verify_winrm_plugin
             " Please upgrade to version 1.6 or higher from #{WEBSITE}."
         end
 
-        if !winrm_plugin_installed?
+        if Gem::Version.new(vagrant_version) < Gem::Version.new("2.1.5") && !winrm_plugin_installed?
```

Comment:
`<= Gem::Version("2.1.5")` or `< Gem::Version("2.1.6")`

Reply:
Thanks @chrisroberts ! I meant to message yesterday - I was testing this locally from TK and you obviously already saw this PR 😄

Reply:
@chrisroberts Do you have an ETA for 2.1.6 yet?

Reply:
@tyler-ball Next release will be 2.2.0 and we are shooting to have it wrapped up and ready to go tomorrow. If things aren't fully ready for tomorrow then it will be Tuesday next week.

Reply:
Need to fix this check to be `< 2.2.0`